CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 ### Added
 - Python 3.10 support
 
+### Changed
+- The phone capability moved to a plugins folder, so if you use that capability
+  and want to keep it, you should add `two_factor.plugins.phonenumber` line in
+  your `INSTALLED_APPS` setting.
```

Comment:
We should also note that the templatetag library has been changed from "two_factor" to "phonenumber"